repository-wildfly/src/main/webapp/WEB-INF/jboss-deployment-structure.xml

```diff
@@ -0,0 +1,10 @@
+<jboss-deployment-structure>
+  <deployment>
+    <exclusions>
+      <module name="org.jboss.resteasy.resteasy-jackson-provider"/>
+    </exclusions>
+    <dependencies>
+      <module name="org.jboss.resteasy.resteasy-jackson2-provider" services="import"/>
```

Comment:
why do we need this? I am always reluctant to excluding modules and sort of replacing defaults...

Reply:
i thought we do to tell wildfly to use jackson 2 and not 1 but seems i was wrong (it already does that by default). will remove - nice catch thank you